src/main.py

```diff
@@ -30,14 +31,66 @@ def handle_invalid_usage(error):
 def sitemap():
     return generate_sitemap(app)
 
-@app.route('/user', methods=['GET'])
-def handle_hello():
+@app.route('/users/<int:id>', methods=['GET'])
+def get_user(id):
+    if User.get_user(id) is None:
```

Comment:
```suggestion
    if not User.get_user(id):
```
Cuando se comprueba que algo existe en vez de comparar con `None` es más común y legible usar la característica de los objetos y listas de que si son `None` o están vacíos se evalúan como `False`